CODE_OF_CONDUCT.md

```diff
@@ -77,8 +77,8 @@ available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.ht
 For answers to common questions about this code of conduct, see
 https://www.contributor-covenant.org/faq
 
-## Don't hesitate to read [our documentation](https://docs.luos.io), or to post your questions/issues on the [Luos' Forum](https://community.luos.io). :books:
+## Don't hesitate to read [our documentation](https://www.luos.io), or to post your questions/issues on the [Luos' Forum](https://community.luos.io). :books:
```

Comment:
We should update all links mentioning our old 'Forum' and replace the links https://community.luos.io by https://www.luos.io/community (Redirection to the Discord)